browser.js

```diff
@@ -7,10 +7,11 @@ export default async function startBrowser() {
     browser = await puppeteer.launch({
       headless: false,
       args: ["--disable-setuid-sandbox"],
-      'ignoreHTTPSErrors': true
+      'ignoreHTTPSErrors': true,
+      slowMo: 200,
```

Comment:
Added this in because one of the sites seemed to know we were using an automated browser and blocked us.